interfaces/IDeviceInfo.h

```diff
@@ -25,136 +25,315 @@
 
 namespace Thunder {
 namespace Exchange {
-
+    /* @json 1.0.0 */
```

Comment:
I think you can drop the annotation  ```/* @json 1.0.0 */``` here

Reply:
as per the discussion it is still required to make sure jsonrpc interfaces with backward compatible